x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/columns.tsx

```diff
@@ -133,6 +135,14 @@ export const progressColumn = {
   'data-test-subj': 'mlAnalyticsTableColumnProgress',
 };
 
+export const getDFAnalyticsJobIdLink = (item: DataFrameAnalyticsListRow) => {
+  return (
+    <EuiLink href={getResultsUrl(item.id, getAnalysisType(item.config.analysis))}>
```

Comment:
We need to mirror the behavior of the links from the IDs in the management page Anomaly detection jobs list, which open the jobs list page in the ML plugin, rather than linking to the results page here.

You'll need to add in functionality for checking a job ID passed in the URL, and if present set the job ID in the query bar of the jobs list e.g.

![image](https://user-images.githubusercontent.com/7405507/81560784-006af180-938a-11ea-8dd7-73ac108950e3.png)

Reply:
Hi Pete, this is now fixed in https://github.com/elastic/kibana/pull/65778/commits/1ec4fde438331af6d9bdb62f1ffe80563ca3e110